salt/containerd/installed.sls

```diff
@@ -0,0 +1,24 @@
+{%- from "containerd/map.jinja" import containerd with context %}
+{%- from "repo/map.jinja" import repo with context %}
+
+include:
+  - repo
+  - runc
+
+Install containerd:
+  pkg.installed:
+    - name: containerd
+    - version: {{ containerd.version }}
+    - fromrepo: {{ repo.containerd.name }}
```

Comment:
So, this means we keep the *version* of `containerd` we want to install, and the *repository* from where to retrieve it, are managed in separate locations. Is that a wise approach? Would it make sense to combine these in one place?

Reply:
Just wanted to separate repo stuff from real program stuff but maybe version should be part of the repo, make sense.